SingularityUI/app/components/common/modalButtons/NewDisabledActionModal.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import { FetchDisabledActions, NewDisabledAction } from '../../../actions/api/disabledActions';
+
+import FormModal from '../modal/FormModal';
+
+const DISABLED_ACTION_TYPES = ['BOUNCE', 'DEPLOY', 'SCALE', 'REMOVE', 'DECOMMISSION'];
+
+import Utils from '../../../utils';
+
+class DeleteDisabledActionModal extends Component {
+  static propTypes = {
+    user: PropTypes.string,
+    newDisabledAction: PropTypes.func.isRequired
+  };
+
+  show() {
+    this.refs.newDisabledActionModal.show();
+  }
+
+  render() {
+    return (
+      <FormModal
+        ref="newDisabledActionModal"
+        name="New Disabled Action"
+        action="Create Disabled Action"
+        buttonStyle="success"
+        onConfirm={(data) => this.props.newDisabledAction(data.type, data.message)}
+        formElements={[
+          {
+            type: FormModal.INPUT_TYPES.SELECT,
+            name: 'type',
+            label: 'Type',
+            isRequired: true,
+            options: DISABLED_ACTION_TYPES.map((type) => ({
```

Comment:
Yeah, I didn't write the backend to support multiple yet, might be a good idea at some point. However, it does support overriding the message for one that is already present (i.e if you issue a new request for disabling bounces when they are alreayd disabled, it will take the message form the newer one). So I might leave it be for the moment